### PR TITLE
MEN-5477: mender-gateway: install inventory script

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
These go together:
* https://github.com/mendersoftware/mender-gateway/pull/16
* https://github.com/mendersoftware/meta-mender/pull/1608
* https://github.com/mendersoftware/mender-dist-packages/pull/210

**Comment by lluiscampos** [comment]
Build @ [Pipeline 481988120](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/481988120)

File: tests/acceptance/commercial/test_mender_gateway.py [hunk 1]
```diff
@@ -44,7 +44,11 @@ def test_build_mender_gateway(
             request, prepared_test_build["build_dir"], "core-image*.ext4"
         )
 
-        output = subprocess.check_output(
-            ["debugfs", "-R", "stat /usr/bin/mender-gateway", image]
-        ).decode()
-        assert "Type: regular" in output
+        for file in [
```
Comment (oleorhagen) [inline tests/acceptance/commercial/test_mender_gateway.py:47]: Super nit:
It is customary to  use `(tuples)` for constant use

**Review by oleorhagen — APPROVED** [review]

File: tests/acceptance/commercial/test_mender_gateway.py (hunk 1 above)
Comment (lluiscampos) [reply tests/acceptance/commercial/test_mender_gateway.py]: Fixed also for `test_monitor_addon.py`

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline :snail:

**Pull request merged**